packages/cozy-sharing/src/components/Recipient.jsx

```diff
@@ -295,15 +298,83 @@ const Status = ({ status, isMe, instance }) => {
   )
 }
 
+/**
+ * Displays the confirmation interface that allows to confirm or reject a recipient
+ */
+const RecipientConfirm = ({
+  recipientConfirmationData,
+  rejectRecipient,
+  confirmRecipient
+}) => {
+  const { t } = useI18n()
+
+  const reject = () => {
+    rejectRecipient(recipientConfirmationData)
+  }
+
+  const confirm = () => {
+    confirmRecipient(recipientConfirmationData)
+  }
+
+  return (
+    <>
+      <Chip
+        className={cx(styles['recipient-confirm__action--confirm'])}
+        variant="outlined"
+        size="small"
+        onClick={confirm}
+      >
+        <Icon icon={CheckCircleIcon} style={{ marginRight: '0.5rem' }} />
```

Comment:
prefer the use of classname={'u-mr-half'}

Reply:
Fixed